app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:layout_marginTop="12dp"
+  android:layout_marginBottom="@dimen/activity_margin_horizontal"
+  android:layout_marginStart="@dimen/activity_margin_horizontal"
+  android:layout_marginEnd="@dimen/activity_margin_horizontal"
+  android:layout_marginLeft="@dimen/activity_margin_horizontal"
+  android:layout_marginRight="@dimen/activity_margin_horizontal"
+  android:elevation="6dp">
+
+  <LinearLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:weightSum="8">
+    android:orientation="vertical">
 
-    <androidx.appcompat.widget.AppCompatSpinner
+    <RelativeLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="10dp"
+      android:layout_marginBottom="10dp"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp">
+
+      <ImageView
+        android:id="@+id/btn_remove"
+        android:layout_width="24dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:layout_alignParentEnd="true"
+        android:contentDescription="@string/remove_description"
+        android:visibility="visible"
+        app:srcCompat="@drawable/ic_clear_24dp" />
+
+      <androidx.appcompat.widget.AppCompatSpinner
```

Comment:
Replace this with `Spinner` we rarely need to reference `AppCompat*` but it is good to check the docs

```
/**
 * A {@link Spinner} which supports compatible features on older versions of the platform,
 * including:
 * <ul>
 *     <li>Allows dynamic tint of its background via the background tint methods in
 *     {@link androidx.core.view.ViewCompat}.</li>
 *     <li>Allows setting of the background tint using {@link R.attr#buttonTint} and
 *     {@link R.attr#buttonTintMode}.</li>
 *     <li>Setting the popup theme using {@link R.attr#popupTheme}.</li>
 * </ul>
 *
 * <p>This will automatically be used when you use {@link Spinner} in your layouts.
 * You should only need to manually use this class when writing custom views.</p>
 */
public class AppCompatSpinner extends Spinner implements TintableBackgroundView {
```

Reply:
Done!